api/overview/azure/batch.md

```diff
@@ -29,12 +37,44 @@ Install-Package Azure.Batch
 dotnet add package Azure.Batch

```

Comment:
Ok. Could you add "NET Core command line" over this command; similar to how it is [here](https://docs.microsoft.com/en-us/dotnet/api/overview/azure/sql?view=azure-dotnet)